README.md

```diff
@@ -1,5 +1,7 @@
 # Introduction
```

Comment:
At installation: change link from `git clone --recursive https://github.com/thorstenhater/arbor-gui.git` to `https://github.com/arbor-sim/gui.git`

Reply:
Yes, I wish I could use `*this` here.